### PR TITLE
emulated_hue: add documentation for new entities configuration

### PR DESCRIPTION
**Pull request by jackwilsdon**
**Description:** Add documentation for new `entities` section in the `emulated_hue` component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11981

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/emulated_hue.markdown [hunk 1]
```diff
@@ -90,26 +92,17 @@ emulated_hue:
...
-    light.office_light:
-      # Address light.office_light as "back office light"
-      emulated_hue_name: "back office light"
+      name: "Bedside Lamp"
+    light.ceiling_lights:
+      hidden: true
 ```
 
-The following are attributes that can be applied in the `customize` section:
+The following are attributes that can be applied in the `entities` section:
```
Comment (jackwilsdon) [inline source/_components/emulated_hue.markdown]: Should we keep the old `customize` section but mention that the `entities` attribute is the preferred way of doing things?

Comment (balloob) [reply source/_components/emulated_hue.markdown]: Yeah, let's add a single line at the bottom that this used to be part of customize but that we're moving to have config under `entitities`

**Comment by frenck** [comment]
@jackwilsdon The parent PR has been merged. Could you please handle the last bit of the documentation so it can be merged as well? 👍

**Comment by jackwilsdon** [comment]
@frenck added a note regarding moving and future deprecation.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
@jackwilsdon Thank you for the quick action. I've taken the liberty to add some more small changes to improve the page overall.

Can be merged as soon as the build passes.

**Pull request merged**